.rubocop_todo.yml

```diff
@@ -140,6 +133,7 @@ RSpec/NoExpectationExample:
     - 'spec/unit/puppet/provider/iis_virtual_directory/webadministration_spec.rb'
     - 'spec/unit/puppet/type/iis_site_spec.rb'
     - 'spec/unit/puppet/type/iis_virtual_directory_spec.rb'
+    - 'spec/acceptance/iis_virtual_directory_test_spec.rb'
```

Comment:
we should remove `'spec/acceptance/iis_virtual_directory_spec_test.rb` from above list as well?

Reply:
This will remain here because for each expect block we have a method written instead of explicitly calling expect inside each it block of specs.